README.md

```diff
@@ -17,21 +17,46 @@ A super simple Java project that utilizes DALL-E3, LangChain4J, and Azure Open A
     ```bash
     git clone https://github.com/sandraahlgrimm/langchain4j-dalle3-demo.git
     ```
+    
```

Comment:
Let's remove this as the latest main should always be tested and working

Reply:
I will abide. Of course the latest main should always be tested and working. In practice I have observed numerous bits of guidance where I, as a reader, ask the question, "did the author make changes to the code but forget to update the guidance? Can I please see a version of the guidance that matches the code I'm actually seeing. Or vice versa."

Reply:
Let's put it at the end then. The first glance at the README should introduce the project easily.